fibonacci/bb/code.clj

```diff
@@ -0,0 +1,14 @@
+(defn- fibonacci [n]
+  (if (<= n 1)
```

Comment:
Can we make this two separate if conditons please?

Reply:
Done.